docs/_guide/decorators.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: guide
+title: Using decorators
+slug: decorators
+---
+
+Decorators are special declarations that you can use when defining classes, class methods, and class fields. LitElement supplies a set of decorators that reduce the amount of boilerplate code you need to write when defining a component. 
+
+For example, the `customElement` and `property` decorators make a basic element definition more compact:
+
+```js
+import {LitElement, html, customElement, property} from 'lit-element';
+
+@customElement('my-element')
+class MyElement extends LitElement {
+
+ // Declare observed properties
+ @property()
+ adjective = 'awesome';
+
+ // Define the element's template
```

Comment:
As in, "Declare the element's template"?

Reply:
Oh yeah... I was reading this as the comment on the render method, ie "Declare the element's render method". I've never really liked the "define the template" terminology because it doesn't correspond to anything super concrete. A render method is usually composed of several templates, and doesn't even need to return a TemplateResult technically.

Reply:
Hm... I get where you're going, but the current docs are all kind of written this way. We have a "Templates" chapter, not a "Render" chapter.  Suggest we discuss this outside the PR, maybe make a larger change to the docs structure later?

Reply:
Sounds good